tools/SetupFlow/DevHome.SetupFlow.ElevatedServer/DevHome.SetupFlow.ElevatedServer.csproj

```diff
@@ -20,16 +20,6 @@
   </ItemGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\DevHome.SetupFlow.Common\DevHome.SetupFlow.Common.csproj" />
-    <ProjectReference Include="..\DevHome.SetupFlow.ElevatedComponent\DevHome.SetupFlow.ElevatedComponent.csproj" />
-  </ItemGroup>
-
-  <!-- The server .exe requires the .winmd to be located next to it. The project reference 
-        is not enough to have it automatically placed there. Manually include it in the outputs. -->
-  <ItemGroup>
-    <Content Include="..\DevHome.SetupFlow.ElevatedComponent\bin\$(Platform)\$(Configuration)\$(TargetFramework)\DevHome.SetupFlow.ElevatedComponent.winmd">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-      <CopyToPublishDirectory>Always</CopyToPublishDirectory>
-    </Content>
+    <ProjectReference Include="..\DevHome.SetupFlow\DevHome.SetupFlow.csproj" />

```

Comment:
I'm not a fan of having the elevated process depend on the full SetupFlow project. That makes the elevated server depend on the whole setup flow, which depends on the elevated server. It's not an issue for the compiler, but I'm still not comfortable with that.
I would put the shared parts in the Common project, and keep there only the minimum we need. 

But if we're going to keep the dependency on the SetupFlow, I think we can do away with the Common project altogether